### PR TITLE
Feat(#22): add django models

### LINKED ISSUE #22: Model all data required for TMLP and create entities in the ORM
e.g. Trees Configuration, Trees, Build status, Test status, Failures

### PR DESCRIPTION
**Pull request by mari1912**
## Description

- Add django models
- Add connection to database, it needs to be running on the background
- fix backend pre-commit
- todo: connect the database to docker-compose

## Related Issues
- Closes #22 

## How to test it
- Add a .env file by running a `cp .env.example .env`
- Fill the values with kcidb user/password
- Run kcidb_playground in the background
- Try to run `poetry run python3 manage.py inspectdb` to check if the app can get the db values
- Run `poetry run python3 manage.py shell` and inside the shell 
```python
from kernelCI_app.models import Issues
x = Issues.objects.all()
print(x)
```
and check if there is any issue object

### CODE REVIEW

File: backend/kernelCI/settings.py [hunk 1]
```diff
@@ -27,6 +29,8 @@
 
 ALLOWED_HOSTS = []
 
+env = environ.Env(DEBUG=(bool, False))
```
Comment (barbieri) [inline backend/kernelCI/settings.py:32]: we should be using `get_json_env_var()` https://github.com/profusion/kernelCI-dashboard/blob/main/backend/kernelCI/settings.py#L18C5-L18C21

File: backend/kernelCI/settings.py [hunk 2]
```diff
@@ -27,6 +29,8 @@
...
+environ.Env.read_env(os.path.join(BASE_DIR, '.env'))
```
Comment (lfjnascimento) [inline backend/kernelCI/settings.py:33]: update your branch and than you can use `get_json_env_var`

File: backend/kernelCI/settings.py (hunk 1 above)
Comment (lfjnascimento) [reply backend/kernelCI/settings.py:32]: ust rebase your branch and you can use `get_json_env_var`

File: backend/kernelCI/settings.py (hunk 2 above)
Comment (barbieri) [inline backend/kernelCI/settings.py:33]: this is never to be used from within the code, but rather from some external env loader. Be it a `source .env` in the shell that calls the server, https://pypi.org/project/dotenv/ with a forced import before it runs... 

anyway, never to be done/called-directly in the project code

**Review by lfjnascimento — APPROVED** [review]

**Pull request merged**